recipes-kernel/linux/linux-%.bbappend

```diff
@@ -1 +1,4 @@
-require linux-readonly-rootfs-overlay.inc
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
```

Comment:
@DaniilKl do we know why using .scc is not working with sunxi build?

Reply:
@TomaszAIR I found line `git://git.yoctoproject.org/yocto-kernel-cache;type=kmeta;name=meta;branch=yocto-5.15;destsuffix=${KMETA};protocol=https` only in variables `SRC_URI` which are declared in recipes `linux-yocto...`. So it seems that `linux-mainline` does not use  `yocto-kernel-cache`

Reply:
It can be checked using `bitbake-getvar -r linux-mainline SRC_URI`

Reply:
@DaniilKl so let's then add linux-mainline_%.bbappend with that change and in out meta layer from the project make sure that we remove this KERNEL_FEATURE:append so it will not make build fail

Reply:
@TomaszAIR You mean add `linux-mainline_%.bbappend` into `meta-readonly-rootfs-overlay` using this PR?

Reply:
@DaniilKl Yes, your change should be added as `linux-mainline_%.bbappend` and in out layer should remove KERNEL_FEATURE:append from `linux-%.bbappend` so it will build correctly

Reply:
@TomaszAIR Amended in last two commits

Reply:
@DaniilKl in meta-readonly-rootfs-overlay we should leave both linux-%.bbappend and linux-mainline_%.bbappend.

Then in our custom (project specific layer) we should add `KERNEL_FEATURE:remove = " features/overlayfs/overlayfs.scc"`. This way we will still support every linux that uses linux-yocto by using .scc file and also support linux-mainline by adding needed .cfg file.

Even better would be to add `KERNEL_FEATURE:remove = " features/overlayfs/overlayfs.scc"` to linux-mainline_%.bbappend in this layer - there will be no need for changes in any project specific layer.

Reply:
@TomaszAIR Done 2c3114bb65beac8f13dbe0805295e2e3ec942a6f